### ISSUE TITLE
Replace thick “Open file” icon with thin “Open file” icon

### ISSUE THREAD

**Issue by Patricek0056** [black7375/Firefox-UI-Fix#775]
### Is your feature request related to a problem?

Mozilla updated “Open file” icon in library, but not in toolbar.

### The solution I'd like

So, my suggestion is to replace the old thick “Open file” icon with a new thin “Open file” icon.

Like in #553 

#### Old icon in toolbar:

![Old open file icon](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/7d57521e-21eb-407e-ab65-43ccace915df)

#### New updated icon in library:

![New open file icon](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/29731d8e-81a4-4aaa-977d-34db4fa035e2)

In short, the icon from the second screenshot should replace the icon from the first screenshot.

**Comment by black7375** [comment]
Patch is reflected!!
![image](https://github.com/black7375/Firefox-UI-Fix/assets/25581533/80da69e0-6793-4f91-ba23-6f46bb2fc2ad)
